docs/sp-migration/generate-manifests.sh

```diff
@@ -1,13 +1,25 @@
 #!/bin/sh
 set -e
 
+# shellcheck disable=SC2034
 version_aws=v0.37.0
+# shellcheck disable=SC2034
 version_azure=v0.34.0
+# shellcheck disable=SC2034
 version_gcp=v0.34.0
 
 rm -f "sp-manual.yaml" && touch "sp-manual.yaml"
 rm -f "sp-family-manual.yaml" && touch "sp-family-manual.yaml"
-kubectl get managed --no-headers -o jsonpath='{range .items[*]}{.apiVersion}{"\n"}{end}' | grep -E '(aws|gcp|azure).upbound.io' | sort | uniq | while read -r line
+
+if [ -n "$CONF_PATH" ]; then
+  echo "Generating manifests from $CONF_PATH"
+  apiGroups=$(grep -rh apiVersion: "$CONF_PATH" | grep -E '(aws|gcp|azure).upbound.io' | sort | uniq | tr -d '[:blank:]'| cut -d ":" -f 2)
+else
```

Comment:
Hey @ytsarev, there might be some MRs created with monolith providers that do not belong to any configuration/composition. Therefore, getting the existing MRs from the cluster is safer before removing monolith providers.

Reply:
Sure, but there might be cases when I want to migrate Composition within the Configuration code with API migrator and in a fully offline mode and then generate the Provider installation manifests. Both options are not mutually exclusive.